templates/analytics/support.html

```diff
@@ -18,11 +18,21 @@
         </center>
     </form>
 
-    {% if message %}
+
+    {% if error_msg %}
+    <div class="alert alert-danger">
+        <center>
+            {{ error_msg }}
+        </center>
+    </div>
+    {% elif message %}
     <div class="alert alert-success">
         <center>
             {{ message }}
         </center>
+        {% if redirect_to %}
```

Comment:
Don't keep the meta tag here. It should go `<HEAD>`.

So create a customhead block and put it inside that. You can place the block after the title block.